### PR TITLE
AdminCRUD #11 to Master

### PR DESCRIPTION
**Pull request by BadBrahmin**

### CODE REVIEW

**Comment by yashjk** [comment]
Approved!

**Comment by AmanSharma4419** [comment]
Approved

File: controllers/newContent.controller.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    .catch(err => {
+      res.status(500).send({
+        message: err.message || "Some error occurred while retrieving content."
+      });
+    });
+};
+
+//////////////////////
+// Find a single content with a contentId
+exports.findOne = (req, res) => {
```
Comment (prank7) [inline controllers/newContent.controller.js]: Don't export single functions. Export them together in a module pattern.

`module.exports = {
    fn1,
    fn2
}`

File: controllers/newContent.controller.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
+const Content = require("../Models/contentSchema");
```
Comment (prank7) [inline controllers/newContent.controller.js]: The file name should just be named `content.controller.js` and not `newContent.controller.js`

**Review by prank7 — CHANGES_REQUESTED** [review]

File: controllers/newContent.controller.js (hunk 1 above)
Comment (BadBrahmin) [reply controllers/newContent.controller.js]: Resolved.

**Comment by BadBrahmin** [comment]
@prank7  Changes done.

**Review by prank7 — APPROVED** [review]

**Pull request merged**